js/stores/eventStore.js

```diff
@@ -128,6 +139,9 @@ const doAction = (action) => {
       // retains all past pages, not really sure that's needed... [MTR]
       eventState = eventState.set('page_info', eventState.get('page_info').push(action.pageInfo))
       break
+    case WindowConstants.WINDOW_GOT_RESPONSE_DETAILS:
+      addResponseDetails(action.frameProps.get('location'), action.frameProps.get('tabId'), action.details)
```

Comment:
as we discussed, I think we should just use this for the load event and add the details to it